### ISSUE TITLE
Error with escaped character

### ISSUE THREAD

**Issue by shiyao-afterpay** [actgardner/gogen-avro#164]
The following fails:
example.avsc
```
{
  "type" : "record",
  "name" : "foobar",
  "fields" : [ {
    "name" : "foobar",
    "type" : "long",
    "doc" : "\n,",
    "default" : 0
  }]
}
```
gogen-avro output example.avsc

Error writing source files to directory "output" - Error writing file output/foobar.go - 18:1: expected '}', found ',' (and 1 more errors)

Problem occurs due to the \n escaped char.  This example.avsc is generated from the avdl with the official java tools. So it shall be valid itself.

**Comment by shiyao-afterpay** [comment]
@actgardner thoughts on this?

**Comment by actgardner** [comment]
@shiyao-afterpay Thanks for reporting this issue! It should be fixed in the version I just released (v7.3.1)
